tests/test_impl_loader.py

```diff
@@ -0,0 +1,21 @@
+import os
+import unittest
+
+from test_relative_import.relative_import_class import Sample
+from getgauge.impl_loader import update_step_resgistry_with_class
+
+
+class ImplLoaderTest(unittest.TestCase):
+    def setUp(self):
+        self.relative_file_path = os.path.join('..', 'test_relative_import', 'relative_import_class.py')
+
+    def test_update_step_resgistry_with_class(self):
+        curr_dir = os.getcwd()
+        os.chdir('tests')
+        method_list = update_step_resgistry_with_class(Sample(), self.relative_file_path)
```

Comment:
```suggestion
        method_list = update_step_registry_with_class(Sample(), self.relative_file_path)
```

I understand this is not related to you change but will be a good time to fix the typo in the test and the implementation, if that's ok.

Reply:
Sure sure, will do it 😄